tests/fpga/pipeline_scope.py

```diff
@@ -43,8 +43,8 @@ def make_sdfg(dtype, name="pipeline_test"):
     post_write = post_state.add_write("b")
     post_state.add_memlet_path(post_read,
                                post_write,
-                               memlet=dace.Memlet.simple(post_write,
```

Comment:
Could we take this opportunity to get rid of `Memlet.simple` in this sample? :-)